spec/controllers/users/omniauth_callbacks_controller_spec.rb

```diff
@@ -309,7 +309,7 @@
           expect(redirect_params["state"]).to eq "1234"
           expect(redirect_params["scope"]).to eq "ial2"
           expect(redirect_params["response_type"]).to eq "code"
-          expect(redirect_params["redirect_uri"]).to eq "http://test.host/users/auth/idme/callback?spouse=true"
```

Comment:
we don't need to include the param on our callback url, only for the "authorize" redirect to ID.me

Reply:
Did you add this expectation because ID.me automatically includes it?

Reply:
lol yes sorry